### PR TITLE
[AFM-214]: Add a new function to the ada embed api called closeCampaign() 

### PR DESCRIPTION
**Pull request by GuelorEmanuel**
Added a new function called `closeCampaign() `  to the ada embed api action section
Screen Shot 2022-02-22 at 4 54 12 PM

### CODE REVIEW

File: source/includes/_embed2.md.erb [hunk 1]
```diff
@@ -375,6 +375,14 @@ When called, the `zdChatterAuthCallback` can be used to request a JWT token from
 ## Actions
 Actions can be called from the global adaEmbed object. A list of the available actions is available below.
 
+### closeCampaign
+`closeCampaign: Promise<void>;`
+
+Can be used to programatically close campaign.
+
+### triggerCampaign
```
Comment (andreirtaylor) [inline source/includes/_embed2.md.erb:383]: Why did you add trigger campaign here?

File: source/includes/_embed2.md.erb [hunk 2]
```diff
@@ -375,6 +375,14 @@ When called, the `zdChatterAuthCallback` can be used to request a JWT token from
 ## Actions
 Actions can be called from the global adaEmbed object. A list of the available actions is available below.
 
+### closeCampaign
+`closeCampaign: Promise<void>;`
```
Comment (ebumbulis) [inline source/includes/_embed2.md.erb:379]: should be `closeCampaign()` since it's a function

File: source/includes/_embed2.md.erb [hunk 3]
```diff
@@ -375,6 +375,14 @@ When called, the `zdChatterAuthCallback` can be used to request a JWT token from
...
+
+Can be used to programatically close campaign.
```
Comment (ebumbulis) [inline source/includes/_embed2.md.erb:381]: I suggest something like: "Closes the currently-displayed campaign (does nothing if no campaign is currently displayed)."

File: source/includes/_embed2.md.erb [hunk 4]
```diff
@@ -375,6 +375,11 @@ When called, the `zdChatterAuthCallback` can be used to request a JWT token from
 ## Actions
 Actions can be called from the global adaEmbed object. A list of the available actions is available below.
 
+### closeCampaign
+`closeCampaign(): Promise<void>;`
```
Comment (andreirtaylor) [inline source/includes/_embed2.md.erb:379]: We need the disclaimer that this is only for engage clients.

Comment (lozog) [reply source/includes/_embed2.md.erb:379]: @GuelorEmanuel ☝️ 
i think Andrei's talking about the disclaimer from line 530

**Review by lozog — CHANGES_REQUESTED** [review]

File: source/includes/_embed2.md.erb (hunk 4 above)
Comment (GuelorEmanuel) [reply source/includes/_embed2.md.erb:379]: I see, thanks for pointing that out.

**Review by lozog — APPROVED** [review]
i guess this should only get merged once the implementation is fully deployed (the `chat` PR is a requirement too, yes?)

**Comment by GuelorEmanuel** [comment]
> i guess this should only get merged once the implementation is fully deployed (the `chat` PR is a requirement too, yes?)

Yes, It was deployed yesterday.
